flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -151,6 +151,16 @@
             "Selects type of sequenced executor used for feeding, valid values are LATENCY, ADAPTIVE, THROUGHPUT",
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
+    public static final UnboundStringFlag RESPONSE_SEQUENCER_TYPE = defineStringFlag(
+            "response-sequencer-type", "ADAPTIVE",
+            "Selects type of sequenced executor used for mbus responses, valid values are LATENCY, ADAPTIVE, THROUGHPUT",
+            "Takes effect at redeployment",
+            ZONE_ID, APPLICATION_ID);
+    public static final UnboundIntFlag RESPONSE_NUM_THREADS = defineIntFlag(
+            "response-num-threads", 2,
+            "Selects type of sequenced executor used for mbus responses, valid values are LATENCY, ADAPTIVE, THROUGHPUT",
```

Comment:
Duplicated description of added flag above

Reply:
Thanks, Fixed